### ISSUE TITLE
`cabal-rpm-0.13.3 local lens-family`: getDirectoryContents:openDirStream: does not exist (No such file or directory)

### ISSUE THREAD

**Issue by refi64** [juhp/cabal-rpm#59]
```
$ cabal-rpm lens-family
Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.IVsD8k
+ umask 022
+ cd /home/ryan/rpmbuild/BUILD
+ cd /home/ryan/rpmbuild/BUILD
+ rm -rf lens-family-1.2.3
+ /usr/bin/gzip -dc /home/ryan/rpmbuild/SOURCES/lens-family-1.2.3.tar.gz
+ /usr/bin/tar -xof -
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd lens-family-1.2.3
+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ exit 0
cabal-rpm: lens-family-1.2.3: getDirectoryContents:openDirStream: does not exist (No such file or directory)
$ cabal-rpm --version
0.13.3
$ 
```

**Comment by juhp** [comment]
Just checking: you ran `cabal-rpm local lens-family`?

**Comment by juhp** [comment]
I was able to do the following on Fedora 30:

```
$ podman run -it --rm fedora:30
# dnf install -y cabal-rpm
# cabal-rpm builddep lens-family
# cabal-rpm install lens-family
```

**Comment by refi64** [comment]
Ah sorry, I screwed up my reproducer. This should reproduce the issue inside a vanilla Fedora 30 container:

```
$ dnf install -y cabal-rpm
$ cabal-rpm spec lens-family
$ cabal-rpm local lens-family
```

Future runs of `cabal-rpm spec` will show the same issue. I just screwed up the command...

**Comment by juhp** [comment]
Oops, sorry for missing your followup...

I suspect this was since fixed - at least I can reproduce with cabal-2.0 for example.
Could you test again and check if it is working for you?

**Comment by juhp** [comment]
Okay, closing this, but do comment or re/open if you still see a problem with newer cabal-rpm, thanks
